docs/jest-integration.md

```diff
@@ -160,6 +212,29 @@ describe("My describe", () => {
 )};
 ```
 
+### TypeScript Jest fuzz tests
+
+To write Jest fuzz tests in TypeScript, the appropriate setup steps
```

Comment:
"To write Jest fuzz tests in TypeScript, the appropriate setup steps
[described above](#enabling-typescript-jest-tests) have to be performed. After
that, the Jest fuzz test can be written in TypeScript as expected. An example
project is available at
[jest_typescript_integration](../examples/jest_typescript_integration)." - can be cut, it repeats the above. Here you can just go on and describe the fuzz test.

Reply:
Rephrased.